src/main/java/duke/Commands/AddInput.java

```diff
@@ -0,0 +1,106 @@
+package duke.Commands;
+
+import duke.Exceptions.*;
+import duke.Storage.Storage;
+import duke.TaskList.TaskList;
+import duke.Tasks.Task;
+import duke.Parser.Parser;
+import duke.Ui.Ui;
+
+import java.util.Scanner;
+
+/**
+ * The AddInput method adds to a given TaskList based on the command given
+ * saves the list of Tasks in the TaskList into a txt file
+ */
+public class AddInput {
+
+    /**
+     * Adds a Task into the given TaskList object based on the given command
+     * @param taskList
+     * @param storage
+     */
+    public static void addInput(TaskList taskList, Storage storage) {
+        Scanner scanner = new Scanner(System.in);
+
+
+        while(scanner.hasNext()){
+            try{
+                String input = scanner.nextLine();
+                //splitting into list for easier comparison
+                String[] inputList = input.trim().split(" ", 2);
+
+                // case where input is bye, and a case where the inputList is of length 1
+                if(Parser.isBye(inputList[0].trim().toLowerCase())){
+                    Ui.byeGreetings();
+                    storage.saveTasks(taskList);
+                    break;
+                }
+
+                // Case where input is list, to show the list of tasks, and case where the inputList is of length 1
+                if(Parser.isList(inputList[0].trim().toLowerCase()) && inputList.length > 1){
```

Comment:
Please put a white space after keywords such as if and while, and also after the closing parenthesis and before the braces.